kubernetes-workshop/6001_basics/modul_basics_6001.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+Name: Kubernetes Basics
+
+Area: Kubernetes Workshop
+
+ID: 6001
+
+Sources:
+    - External_URLs:
+        - https://kubernetes.io/docs/home/
+
+Documents:
+    - Presentation: modul_basics_6000_01_slides.html
+
+Duration: 1h
+
+Description: >
+    This workshop will provide you with a basic introduction to Kubernetes and
+    allows you to deepen you knowledge with future workshops.
+
+Requirements:
+    - Linux Distribution
+    - Bash shell
+    - minikube
+    - kubectl
```

Comment:
We should think about that. But I would recommend to specify a standard and update all YAML files accordingly.